vtex/omnishipping
# Changed leanshipping to always stay disabled id checkout io is installed

#### What is the purpose of this pull request?
### Changed

- `leanShipping` flag to always stay disabled if Checkout IO is installed.

#### What problem is this solving?
Before render extension loader would load runtime and apps so we could not know if checkout IO is installed beforehand.

#### How should this be manually tested?
1. Add [items to cart](https://fernando--checkoutio.myvtex.com/cart/add?&workspace=fernando&sku=31&qty=1&seller=1&sc=1);
2. Proceed to Shipping;
3. Add beta cookie and refresh page;
3. Type postal code `22071060`;
4. LeanShipping should be disabled due to knowing.

#### Screenshots or example usage
n/a
#### Types of changes
